src/patternfly/components/Pagination/pagination.scss

```diff
@@ -0,0 +1,199 @@
+@import "../../patternfly-utilities";
+
+.pf-c-pagination {
+
+  /* stylelint-disable */
+
+  // Base
+  --pf-c-pagination--FontSize:                          var(--pf-global--FontSize--sm);
+
+  // Generic items
+  --pf-c-pagination--item--PaddingLeft:                 var(--pf-global--spacer--lg);
+  --pf-c-pagination--item--PaddingRight:                var(--pf-global--spacer--lg);
+
+  // Select
+  --pf-c-pagination__select--item--MarginRight:         var(--pf-global--spacer--sm);
+
+  // Nav
+  --pf-c-pagination__nav--MinHeight:                    #{pf-size-prem(44px)};
+
+  // --pf-c-pagination__nav--FontSize:                     var(--pf-global--FontSize--lg);
+  --pf-c-pagination__nav--BorderWidth:                  var(--pf-global--BorderWidth--sm);
+  --pf-c-pagination__nav--BorderColor:                  var(--pf-global--BorderColor--light-200);
+
+  // Nav page
+  --pf-c-pagination__nav-page--PaddingRight:            var(--pf-global--spacer--md);
+  --pf-c-pagination__nav-page--PaddingLeft:             var(--pf-global--spacer--md);
+  --pf-c-pagination__nav-page--item--MarginRight:       var(--pf-global--spacer--xs);
+
+  // Dropdown
+  --pf-c-pagination__dropdown__arrow--MarginLeft:       var(--pf-global--spacer--xs);
+
+  // Buttons
+  --pf-c-pagination--button--disabled--Color:           var(--pf-global--disabled-color--200);
+
+  // Complementary text
+  --pf-c-pagination__complementary-text--Color:         var(--pf-global--Color--200);
+
+  // Nav buttons
+  --pf-c-pagination__nav-button--FontSize:              var(--pf-global--FontSize--lg);
+  --pf-c-pagination__nav-button--MinWidth:              #{pf-size-prem(30px)};
```

Comment:
Are these global mins that should come from or be shared with the button component?